samples/serviceinsight/messageviewer/sample.md

```diff
@@ -37,13 +36,13 @@ All the components require, including the views and view models are registered i
 	
 snippet: IoCModule
 
-The plugin must implement the `ICustomMessageBodyViewer` interface that gets invoked when a message is selected. The `Display` method has an argument that is the message object being displayed. Here the message body is decrypted an
+The plugin must implement the `ICustomMessageBodyViewer` interface that gets invoked when a message is selected. The `Display` method has an argument that is the message object being displayed.
 
 snippet: MessageDisplay
 
 ### Endpoint
 
-The endpoint sends a message to a handler, and since the message has sensitive information, it uses a `` object to encrypt the message on the way out, and decrypt it on the way back in. The related classes are in the Shared project so they are shared with the plugin.
+The endpoint sends a message to a handler, and since the message has sensitive information, it encrypts the message on the way out, and decrypt it on the way back in. The related classes are in the Shared project so they are shared with the plugin.
```

Comment:
```suggestion
The endpoint sends a message to a handler, and since the message has sensitive information, it encrypts the message before sending it, and then it is decrypted by the receiver. The related classes are in the Shared project so they are shared with the plugin.
```